src/index.js

```diff
@@ -0,0 +1,28 @@
+import { createTransform } from "redux-persist";
+import LZ from "lz-string";
+import stringify from "json-stringify-safe";
+import process from "process";
```

Comment:
hm, yes that is true, I suppose the best solution here would be to do it like redux https://github.com/reactjs/redux/blob/master/src/combineReducers.js#L5

redux-persist I think needs the same update